journal-letters/modernized/04-chapter-2.adoc

```diff
@@ -37,12 +37,12 @@ and to dwell in that holy fear which the Father thereby placed in my heart.
 Then I came to see what David exhorted to, in the second Psalm,
 when he bid the kings and judges of the earth be wise and learned; and further said,
 "`serve the Lord with fear, and rejoice with trembling.`"
-O! the bowings of my soul! the pleasant dread that dwelt upon my spirit,
+O!--the bowings of my soul! the pleasant dread that dwelt upon my spirit,
 and the reverent tremblings that came over my heart, which filled it with living joy,
 as with marrow and fatness!
 Then could I say in my heart with David, I will wash my hands in innocency,
 and compass your altar, O Lord.
-O! the pleasant drawing near, and that not unprepared,
+O--the pleasant drawing near, and that not unprepared,
```

Comment:
Same here... leave in the ! before the dashes (like you did a few lines up in line 40)